x-pack/test/api_integration/apis/endpoint/policy.ts

```diff
@@ -11,9 +11,13 @@ import { deletePolicyStream } from './data_stream_helper';
 export default function ({ getService }: FtrProviderContext) {
   const esArchiver = getService('esArchiver');
   const supertest = getService('supertest');
+  const ingestManager = getService('ingestManager');
   describe('Endpoint policy api', () => {
     describe('GET /api/endpoint/policy_response', () => {
-      before(async () => await esArchiver.load('endpoint/policy', { useCreate: true }));
+      before(async () => {
+        await ingestManager.setup();
```

Comment:
We can actually remove this because we already do it here: https://github.com/elastic/kibana/blob/master/x-pack/test/api_integration/apis/endpoint/index.ts#L16 before any of the tests run.

Reply:
Good catch. Pretty sure this is left over from me hacking around. I'll remove it